tests/test_input.py

```diff
@@ -1715,7 +1753,7 @@ def test_learning_curve_metrics_and_no_objectives():
     eq_(grid_objectives, [])
```

Comment:
You don't need this `eq_()` statement then?

Reply:
removed